uportal-war/src/main/java/org/jasig/portal/rest/PortletsRESTController.java

```diff
@@ -55,24 +62,22 @@
 @Controller
 public class PortletsRESTController {
 
+    @Autowired
     private IPortletDefinitionRegistry portletDefinitionRegistry;
+
+    @Autowired
     private IPortletCategoryRegistry portletCategoryRegistry;
-    private IPersonManager personManager;
 
     @Autowired
-    public void setPortletDefinitionRegistry(IPortletDefinitionRegistry portletDefinitionRegistry) {
-        this.portletDefinitionRegistry = portletDefinitionRegistry;
-    }
+    private IPersonManager personManager;
 
     @Autowired
-    public void setPersonManager(IPersonManager personManager) {
-        this.personManager = personManager;
-    }
+    private IPortletWindowRegistry portletWindowRegistry;
 
     @Autowired
-    public void setPortletCategoryRegistry(IPortletCategoryRegistry portletCategoryRegistry) {
-        this.portletCategoryRegistry = portletCategoryRegistry;
-    }
+    private IPortletExecutionManager portletExecutionManager;
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
Class is not `final`, so I'd go for `protected` here to make this `Logger` available to putative sub-classes.